### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by chadhietala**

### CODE REVIEW

File: lib/broccoli/glimmer-app.js [hunk 1]
```diff
@@ -215,24 +182,33 @@ class GlimmerApp {
...
     // Remove top-most `src` directory so module names don't include it.
-    const resolvableTree = find(merge([compiledTypeScriptTree, compiledHandlebarsTree]), {
+    const resolvableTree = find( new MergeTrees([compiledTypeScriptTree, compiledHandlebarsTree]), {
       srcDir: 'src'
     });
 
     // Build the file that maps individual modules onto the resolver's specifier
     // keys.
-    const moduleMap = this.buildResolutionMap(resolvableTree);
+    const moduleMap = debug(this.buildResolutionMap(resolvableTree), { name: 'map' });
```
Comment (tomdale) [inline lib/broccoli/glimmer-app.js]: Looks like a debug left in here.

File: package.json [hunk 2]
```diff
@@ -6,43 +6,61 @@
...
+    "babel-plugin-transform-es2015-classes": "^6.23.0",
+    "babel-plugin-transform-es2015-computed-properties": "^6.22.0",
+    "babel-plugin-transform-es2015-destructuring": "^6.23.0",
+    "babel-plugin-transform-es2015-for-of": "^6.23.0",
+    "babel-plugin-transform-es2015-modules-amd": "^6.24.0",
+    "babel-plugin-transform-es2015-parameters": "^6.23.0",
+    "babel-plugin-transform-es2015-shorthand-properties": "^6.22.0",
+    "babel-plugin-transform-es2015-spread": "^6.22.0",
+    "babel-plugin-transform-es2015-template-literals": "^6.22.0",
+    "babel-plugin-transform-proto-to-assign": "^6.23.0",
```
Comment (Turbo87) [inline package.json]: if we add the plugins explicitly should we still depend on `babel-preset-es2015`?

**Comment by dgeb** [comment]
@chadhietala Has this been entirely superseded by more recent work? Or are there changes we should port over in separate PRs?

**Comment by chadhietala** [comment]
Yep

**Comment by chadhietala** [comment]
There is still work to be done but we can now use preprocessor addons.

**Pull request closed without merging**